perf/pulsar-perf-go.go

```diff
@@ -98,12 +99,33 @@ func stopCh() <-chan struct{} {
 
 func RunProfiling(stop <-chan struct{}) {
 	go func() {
-		if err := serveProfiling("0.0.0.0:6060", stop); err != nil && err != http.ErrServerClosed {
+		addr := getIntranetIP()
+		if err := serveProfiling(addr, stop); err != nil && err != http.ErrServerClosed {
 			log.WithError(err).Error("Unable to start debug profiling server")
 		}
 	}()
 }
 
+func getIntranetIP() string {
```

Comment:
Why is this needed? We would still run into a port binding issues when running a producer and consumer right?